readme.txt

```diff
@@ -0,0 +1,13 @@
+# Linear-Regression-Model-for-House-Price-Prediction.

```

Comment:
delete these kind of files which are attached directly in main repo not in your project folder